social_website/management/commands/sync_website.py

```diff
@@ -37,7 +37,7 @@
 from django.core.management.base import BaseCommand, CommandError
 
 from social_website.utils.refresh_stats import refresh_collection_partner_stats, refresh_offline_stats, refresh_online_stats
-from social_website.utils.elastic_search.setup import setup_elastic_search
+# from social_website.utils.elastic_search.setup import setup_elastic_search
```

Comment:
Why this code has been commented? Is this not being used in this file?

Reply:
No sir, i have not commented this code it was commented before only.

Reply:
But it's showing changes in your commit